client/components/jetpack-logo/docs/example.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import JetpackLogo from 'components/jetpack-logo';
+
+export default function JetpackLogoExample() {
+	return (
+		<div>
+			<div>
+				<p>
+					<code>
+						{ '<JetpackLogo full />' }
+					</code>
+				</p>
+				<JetpackLogo full />
```

Comment:
Let's add some `size` here, to demonstrate the `size` prop, and to reduce the size that the example is taking. How about `size={ 100 }`?

Reply:
👍  3fd7a9ecd2a7213d5e4f658378054e0e76fb1778